Retail/src/retail/Retail.java

```diff
@@ -0,0 +1,107 @@
+package retail;
+import java.util.Scanner;
+import retailPackage.RetailItem;
+
+public class Retail {
+
+    /**
+     * Main program that calls the retail class
+     * @param args the command line arguments
+     */
+    public static void main(String[] args) {
+        Scanner keyboard = new Scanner(System.in);
+        
+        //variable init
+        String discription;
```

Comment:
Looks good to me. You do have a spelling error though, it's "description". I tried doing the print methods in the classes for my programs but I couldn't get that to work so I gave up on using them as methods, but it looks good in your program.